test/test_dtype.py

```diff
@@ -249,8 +249,9 @@ def test_uint8_to_int8_overflow(self):
 @unittest.skipIf(Device.DEFAULT == "WEBGL", "No bitcast on WebGL")
 class TestBitCast(unittest.TestCase):
   def test_shape_change_bitcast(self):
-    with self.assertRaises(AssertionError):
-      _test_bitcast(Tensor([100000], dtype=dtypes.float32), dtypes.uint8, [100000])
+    with self.assertRaises(RuntimeError) as cm:
+      _test_bitcast(Tensor.empty((128,), dtype=dtypes.float32), dtypes.uint8)
+    self.assertEqual('shape changing bitcast only supported on DISK right now', str(cm.exception))
```

Comment:
Why? Is this testing functionality?

Reply:
Should i remove all error message checks? I thought they were useful because the code in question can throw for another reason (but that's less relevant here compared to other tests)

Reply:
removed all error message checks